server/xpub-controller/helpers/files.test.js

```diff
@@ -147,41 +147,6 @@ describe('FilesHelper', () => {
     })
   })
 
-  describe('endFileProgress', () => {
```

Comment:
It was necessary and desirable to move these tests out. Using `jest.useFakeTimers()` sometimes caused erratic issues with the other other tests. Moving these out helps with this.